sdk/purview/Azure.Analytics.Purview.Workflows/src/Generated/PurviewWorkflowServiceClient.cs

```diff
@@ -0,0 +1,1277 @@
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> Release a DSAR task request. </summary>
+        /// <param name="taskId"> The task id. </param>
+        /// <param name="content"> The content to send as the body of the request. Details of the request body schema are in the Remarks section below. </param>
+        /// <param name="context"> The request context, which can override default behaviors of the client pipeline on a per-call basis. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="content"/> is null. </exception>
+        /// <exception cref="RequestFailedException"> Service returned a non-success status code. </exception>
+        /// <returns> The response returned from the service. </returns>
+        /// <include file="Docs/PurviewWorkflowServiceClient.xml" path="doc/members/member[@name='ReleaseDsarTaskRequest(Guid,RequestContent,RequestContext)']/*" />
+        public virtual Response ReleaseDsarTaskRequest(Guid taskId, RequestContent content, RequestContext context = null)
+        {
+            Argument.AssertNotNull(content, nameof(content));
+
+            using var scope = ClientDiagnostics.CreateScope("PurviewWorkflowServiceClient.ReleaseDsarTaskRequest");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateReleaseDsarTaskRequestRequest(taskId, content, context);
+                return _pipeline.ProcessMessage(message, context);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> List all workflows. </summary>
+        /// <param name="context"> The request context, which can override default behaviors of the client pipeline on a per-call basis. </param>
+        /// <exception cref="RequestFailedException"> Service returned a non-success status code. </exception>
+        /// <returns> The <see cref="AsyncPageable{T}"/> from the service containing a list of <see cref="BinaryData"/> objects. Details of the body schema for each item in the collection are in the Remarks section below. </returns>
+        /// <include file="Docs/PurviewWorkflowServiceClient.xml" path="doc/members/member[@name='GetWorkflowsAsync(RequestContext)']/*" />
+        public virtual AsyncPageable<BinaryData> GetWorkflowsAsync(RequestContext context = null)
+        {
+            return GetWorkflowsImplementationAsync("PurviewWorkflowServiceClient.GetWorkflows", context);
+        }
+
+        private AsyncPageable<BinaryData> GetWorkflowsImplementationAsync(string diagnosticsScopeName, RequestContext context)
+        {
+            return PageableHelpers.CreateAsyncPageable(CreateEnumerableAsync, ClientDiagnostics, diagnosticsScopeName);
```

Comment:
To resolve build error, you could sync the latest code.

Reply:
Fixed, thanks for your feedback!